lib/vdsm/storage/lvm.py

```diff
@@ -577,74 +577,117 @@ def _getVGDevs(self, vgNames):
                 devices = tuple(devices)
```

Comment:
Note for the commit message - I think you mean:

    Extract helper for reloading vgs